src/components/Table/Row.tsx

```diff
@@ -7,6 +7,11 @@ import { generateCellKey } from './Table.utils'
 import { RowProps } from './types'
 
 export default class Row extends React.PureComponent<RowProps> {
+  handleRowClick = e => {
+    const { row, onRowClick } = this.props
+    onRowClick && onRowClick(row, e)
```

Comment:
Refactor suggestion!

I would add a `defaultProp = { onRowClick: noop }` for this component. That way, `onRowClick` is always there :).

As a user, I also expected the argument order to be `onRowClick(event, row)`. I guess I'm just use to `event` being first? 

😊